packages/logs/lib/models/helpers.ts

```diff
@@ -13,7 +13,7 @@ export interface FormatMessageData {
     environment?: { id: number; name: string } | undefined;
     connection?: { id: number; name: string } | undefined;
     integration?: { id: number; name: string; provider: string } | undefined;
-    syncConfig?: { id: number; name: string } | undefined;
```

Comment:
`syncConfig` is already misnamed since it also applies to `action`. A better name could be simply `script`.
I attempted to rename it but it has a lot of implications including modifying the ES models so I gave up cc @bodinsamuel

Reply:
it was referencing the table which worked with action but indeed does not scale anymore. We can create a new field yes 👍🏻